packages/backend/API.md

```diff
@@ -70,13 +71,18 @@ export { ConstructFactoryGetInstanceProps }
 export { defineAuth }
 
 // @public
-export const defineBackend: <T extends DefineBackendProps>(constructFactories: T) => Backend<T>;
+export const defineBackend: <T extends DefineBackendConstructFactories>(constructFactories: T, props?: DefineBackendProps) => Backend<T>;
 
 // @public (undocumented)
-export type DefineBackendProps = Record<string, ConstructFactory<ResourceProvider & Partial<ResourceAccessAcceptorFactory<never>>>> & {
+export type DefineBackendConstructFactories = Record<string, ConstructFactory<ResourceProvider & Partial<ResourceAccessAcceptorFactory<never>>>> & {
     [K in keyof BackendBase]?: never;
 };
 
+// @public (undocumented)
+export type DefineBackendProps = {
+    mainStackProps: MainStackProps;
+};
+

```

Comment:
I'm sorry I didn't realize that `DefineBackendProps` already exist as a type while providing initial feedback.

We should revert back to existing naming there (otherwise this is breaking change) and name new type something different.

Perhaps `DefineBackendOptions`. @Amplifiyer @edwardfoyle what do you think?

Reply:
To avoid any unnecessary delay, I have changed the new arg to DefineBackendOptions, but can change again easily enough, and DefineBackendConstructFactories has been reverted to DefineBackendProps.